docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -101,7 +102,23 @@ The document is intended for readers with a good knowledge of Linux server manag
 The X-Road Security Server Toolkit package can be installed using PIP:
 
 ```
-$ pip install xrdsst --extra-index-url https://xroad-toolkit.s3-eu-west-1.amazonaws.com/xrdsst
+$ pip install -i http://xroad-toolkit.s3-website-eu-west-1.amazonaws.com/ xrdsst --trusted-host xroad-toolkit.s3-website-eu-west-1.amazonaws.com
```

Comment:
Does it work with ``pip``, it seems it needs to be ``pip3`` always, no?

Reply:
Commented in the UG, that pip or pip3 should be used, whichever is used. For example locally I use "pip3", but on Jenkins it is "pip", it depends on symbolic links, like "python3" could also be run as "python". If I would put only "pip3" there, somebody may ask, why not "pip".